src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
+  margin: 0 auto;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header-left {
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  padding-left: 50px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+}
+
+.nav {
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  gap: 20px;
+  align-items: center;
+  list-style: none;
+  padding: 0;
   margin: 0;
+  height: 100%;
+}
+
+.nav__link.is-active {
+  text-decoration: underline;
+  text-underline-offset: 22px;
+  color: #00acdc;
```

Comment:
Consider using variables for repeating colors

Reply:
fixed